export/export.py

```diff
@@ -0,0 +1,866 @@
+            **kwargs: Additional keyword arguments (not used)
+        """
+        # Store inputs
+        self._edge_program_manager = edge_program
+
+        # Process inputs
+        if self._edge_program_manager is None:
+            raise RuntimeError("Edge program manager is not set.")
+
+        self._executorch_program_manager = self._edge_program_manager.to_executorch(
+            self._backend_config
+        )
+
+    def get_outputs(self) -> ExecutorchProgramManager:
+        """
+        Returns the executorch program manager.
+
+        Returns:
+            The executorch program manager
+
+        Raises:
+            RuntimeError: If the executorch program manager is not initialized
+        """
+        if self._executorch_program_manager is None:
+            raise RuntimeError("Executorch program manager is not initialized.")
+        return self._executorch_program_manager
+
+
+class SourceTransformStage(Stage):
+    """
+    Source transform stage: Apply source transformations to the model.
+    """
+
+    def __init__(self, quantization_recipe: Any) -> None:
+        self._quantization_recipe = quantization_recipe
+        self._transformed_models: Dict[str, nn.Module] = {}
+
+    @property
+    def name(self) -> str:
+        return "source_transform"
+
+    def run(self, models: Dict[str, nn.Module], *args, **kwargs) -> None:
+        """
+        Apply source transformations to the model.
+
+        Args:
+            models: Dictionary mapping method names to PyTorch models
+            **kwargs: Additional keyword arguments (not used)
+        """
+        # Store the original models
+        self._transformed_models = models
+
+        # Check if there's a quantization recipe with ao_base_config
+        if self._quantization_recipe and self._quantization_recipe.ao_base_config:
+            # Apply torchao quantize_ to each model
+            for method_name, model in models.items():
+                for config in self._quantization_recipe.ao_base_config:
+                    quantize_(model, config)
+                    unwrap_tensor_subclass(model)
+                    self._transformed_models[method_name] = model
+
+    def get_outputs(self) -> Dict[str, nn.Module]:
+        """
+        Returns the transformed models.
+
+        Returns:
+            Dictionary mapping method names to transformed models
+        """
+        return self._transformed_models
+
+
+class QuantizeStage(Stage):
+    """
+    Optional stage: Perform post-training quantization on the model.
+    """
+
+    def __init__(self, quantizer: Any) -> None:
+        self._quantizer = quantizer
+        self._quantized_models: Dict[str, nn.Module] = {}
+        self._model_dict: Dict[str, nn.Module] = {}
+        self._exported_models_dict: Dict[str, nn.Module] = {}
+        self._example_inputs_dict: Dict[str, List[tuple[torch.Tensor, ...]]] = {}
+
+    @property
+    def name(self) -> str:
+        return "quantize"
```

Comment:
Would be worth to comment somewhere that quantization may also happen in `SourceTransformStage`